index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+        </div>
+     </section>
+
+     <section id="about-me">
+        <h1>About me</h1>
+        <div class="img3"></div>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me. 
+        </p>
+        <div class="img4"></div>
+        <button class="btngreen">
+            Get My Resume
+        </button>
+     </section>
```

Comment:
- [ ] Kindly adjust the space between this button element and the `background-image` close to it. I would advise you to reposition the button so that it is closer to the `background-image.`

### Your Design

![image](https://user-images.githubusercontent.com/77106962/205453696-6a41a2d0-59c1-4e0c-84ed-84e8476c7e2f.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/205453708-4a9e911d-b35f-483e-b70b-6dac93b86244.png)